modules/starboard/configs/config.json

```
@@ -0,0 +1,209 @@
+{
+	"description": {},
+	"humanName": {
+		"en": "Configuration",
+		"de": "Konfiguration"
+	},
+	"filename": "config.json",
+	"content": [
+		{
+			"name": "channelId",
+			"humanName": {
+				"en": "Starboard channel",
+				"de": "Starboard-Kanal"
+			},
+			"default": {
+				"en": ""
+			},
+			"description": {
+				"en": "In which channel starred messages are sent",
+				"de": "In welchen Kanal gestarrte Nachrichten gesendet werden"
+			},
+			"type": "channelID"
+		},{
+			"name": "emoji",
+			"humanName": {
+				"de": "Emoji"
+			},
+			"default": {
+				"en": "⭐"
+			},
+			"description": {
+				"en": "Which emoji should be used to star messages",
+				"de": "Mit welchem Emoji Nachrichten gestarrt werden sollen"
+			},
+			"type": "string"
+		},{
+			"name": "message",
+			"humanName": {
+				"en": "Message",
+				"de": "Nachricht"
+			},
+			"default": {
+				"en": {
+					"message": "**%stars%** %emoji% in %channelMention%",
+					"color": "#f5c91b",
+					"description": "%content%",
+					"image": "%image%",
+					"author": {
+						"name": "%displayName%",
+						"img": "%userAvatar%",
+						"url": "%link%"
+					}
+				}
+			},
+			"description": {
+				"en": "This message gets send into the selected channel",
+				"de": "Diese Nachricht wird in den ausgewählten Kanal gesendet"
+			},
+			"allowEmbed": true,
+			"type": "string",
+			"params": [
+				{
+					"name": "stars",
+					"description": {
+						"en": "Amount of reactions on the message",
+						"de": "Anzahl der Reaktionen auf die Nachricht"
+					}
+				},{
+					"name": "content",
+					"description": {
+						"en": "The content of the starred message",
+						"de": "Der Inhalt der gestarrten Nachricht"
+					}
+				},{
+					"name": "link",
+					"description": {
+						"en": "A link to the starred message",
+						"de": "Ein Link zur gestarrten Nachricht"
+					}
+				},{
+					"name": "userID",
+					"description": {
+						"en": "The user ID of the author of the starred message",
+						"de": "Die Nutzer-ID des Autors der gestarrten Nachricht"
+					}
+				},{
+					"name": "userName",
+					"description": {
+						"en": "The username of the author of the starred message",
+						"de": "Der Benutzername des Autors der gestarrten Nachricht"
+					}
+				},{
+					"name": "displayName",
+					"description": {
+						"en": "The nickname of the author",
+						"de": "Der Nickname des Autors"
+					}
+				},{
+					"name": "userTag",
+					"description": {
+						"en": "The tag of the author of the starred message",
+						"de": "Der Tag des Autors der gestarrten Nachricht"
+					}
+				},{
+					"name": "userAvatar",
```

```suggestion
					"name": "userAvatar",
					 "isImage": true,
```
`isImage` is needed to allow users to use this parameter in the dashboard as an image